model_card_toolkit/dependencies.py

```diff
@@ -136,7 +125,6 @@ def make_extra_packages_all() -> List[str]:
 def make_required_extra_packages() -> Dict[str, List[str]]:
   """Returns the dict of required extra packages."""
   return {
-      'docs': make_extra_packages_docs(),
```

Comment:
won't this break the TF website generation? Not sure how that is being generated...

Reply:
No, `tensorflow_docs` was never listed as a dependency. Here are the dependencies from 1.3: https://github.com/tensorflow/model-card-toolkit/blob/v1.3.0/setup.py#LL31-L42. It's needed to build the docs locally with `python model-card-toolkit/tools/build_api_docs.py`.

I'm not sure exactly how the docs are being generated for the website yet, e.g. whether internal or tied to a TensorFlow release, but I'm following up on this.

Reply:
Apparently the docs are built internally at Google in a manual process.